test/test_nn.py

```diff
@@ -13520,6 +13520,15 @@ def test_Dropout(self, device):
             input = input.bfloat16()
             self._test_dropout(nn.Dropout, device, input)
 
+    def _test_dropoutNd_no_batch(self, dropout, input):
+        with freeze_rng_state():
+            res_no_batch = dropout(input)
+
+        with freeze_rng_state():
+            res_batched = dropout(input.unsqueeze(0)).squeeze(0)
```

Comment:
tiny nit: I think you could do these both under the same `with freeze_rng_state():`

Reply:
They should be different. Looking at the source, we reset the seed after exiting the context.
https://github.com/pytorch/pytorch/blob/408283319ab6b0eefcc756b6affa3fe7d59db9db/torch/testing/_internal/common_utils.py#L1128-L1144

Also I verified with the following snippet
```python
import torch

from torch.testing._internal.common_utils import freeze_rng_state

with freeze_rng_state():
    print(torch.randn(2))

with freeze_rng_state():
    print(torch.randn(2))
    print("*"*10)

# In same context
with freeze_rng_state():
    print(torch.randn(2))
    print(torch.randn(2))
    print("*"*10)

```

Output
```
tensor([0.0461, 0.4024])
tensor([0.0461, 0.4024])
**********
tensor([0.0461, 0.4024])
tensor([-1.0115,  0.2167])
**********
```

Reply:
Awesome, thanks for clearing that up!